tests/auth_tests/test_hashers.py

```diff
@@ -432,13 +461,12 @@ def test_bcryptsha256_salt_check(self):
     @override_settings(
         PASSWORD_HASHERS=[
             "django.contrib.auth.hashers.PBKDF2PasswordHasher",
-            "django.contrib.auth.hashers.SHA1PasswordHasher",
             "django.contrib.auth.hashers.MD5PasswordHasher",
         ],
     )
     def test_upgrade(self):
         self.assertEqual("pbkdf2_sha256", get_hasher("default").algorithm)
-        for algo in ("sha1", "md5"):
+        for algo in ("md5",):
```

Comment:
Should we remove loop to simplify the test a bit?

Reply:
Maybe we could add `pbkdf2_sha1` in place of `sha1`.